README.md

```diff
@@ -89,6 +89,8 @@
 
 ## Time
 
+[Mozilla](https://hacks.mozilla.org/2019/05/technical-details-on-the-recent-firefox-add-on-outage/). Firefox had an incident in which most add-ons stopped working. The certificates used to sign add-on is expired and involved 15000 addons.
```

Comment:
Hi! I'd like to suggest a more detailed description based on your work.

```suggestion
[Mozilla](https://hacks.mozilla.org/2019/05/technical-details-on-the-recent-firefox-add-on-outage/). Most Firefox add-ons stopped working around May 4th 2019 when a certificate expired. Firefox requires a valid certificate chain to prevent malware. About nine hours later, Mozilla pushed a privileged add-on that injected a valid certificate into Firefox's certificate store, creating a valid chain and unblocking add-ons. This disabled effectively all add-ons, about 15,000, and the resolution took approximately 15-21 hours for most users.
```

The 15-21 (9+6 to 9+12) hours hours comes from the link:
> Starting from about 6 PM Pacific on Friday May 3, we were shipping the fix in Normandy at 2:44 AM, or after less than 9 hours, and then it took another 6-12 hours before most of our users had it.

I would appreciate feedback on this. Despite paring it down, I would especially appreciate suggestions making it more concise.

Other thoughts:

- Should the chain be mentioned at all? This seems to be a trade-off between the length and accuracy of the summary.
- Would "Firefox was later updated to provide fixes to users with push disabled or inoperative." add anything to the description?
- I didn't see any way to include the fact that the problem didn't immediately hit many users or that there was a long tail for resolution.

Reply:
for me seems good as it is now